### PR TITLE
create new exercise raindropds

### PR DESCRIPTION
**Pull request by marianfoo**
create new exercise from https://github.com/exercism/problem-specifications/blob/main/exercises/raindrops/description.md

### CODE REVIEW

File: exercises/practice/raindrops/zcl_raindrops.clas.testclasses.abap [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        methods: test_plang FOR TESTING RAISING cx_static_check.
+        methods: test_plong FOR TESTING RAISING cx_static_check.
+        methods: test_plingplang FOR TESTING RAISING cx_static_check.
+
+ENDCLASS.
+
+CLASS ltcl_raindropds IMPLEMENTATION.
+
+  METHOD setup.
+    cut = NEW ZCL_FIZZ_BUZZ_SOLUTION( ).
```
Comment (larshp) [inline exercises/practice/raindrops/zcl_raindrops.clas.testclasses.abap:18]: ZCL_FIZZ_BUZZ_SOLUTION does not exist

File: exercises/practice/raindrops/zcl_raindrops.clas.testclasses.abap [hunk 2]
```diff
@@ -0,0 +1,52 @@
+CLASS ltcl_raindropds DEFINITION FOR TESTING RISK LEVEL HARMLESS DURATION SHORT FINAL.
+
+  PRIVATE SECTION.
+    DATA cut TYPE REF TO zcl_raindrops.
+    METHODS setup.
+        methods: test_number1 FOR TESTING RAISING cx_static_check.
+        methods: test_number2 FOR TESTING RAISING cx_static_check.
+        methods: test_pling FOR TESTING RAISING cx_static_check.
```
Comment (larshp) [inline exercises/practice/raindrops/zcl_raindrops.clas.testclasses.abap:8]: pretty print

unnecessary chaining

File: exercises/practice/raindrops/zcl_raindrops.clas.testclasses.abap (hunk 1 above)
Comment (marianfoo) [reply exercises/practice/raindrops/zcl_raindrops.clas.testclasses.abap]: fixed

**Comment by larshp** [comment]
CI failing, wrong method name

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**